### ISSUE TITLE
Banking app not supporting ASK as input method

### ISSUE THREAD

**Issue by swssl** [AnySoftKeyboard/AnySoftKeyboard#3435]
### Steps to reproduce
1. Use ASK and the [authentication app](https://play.google.com/store/apps/details?id=com.starfinanz.mobile.android.pushtan) of one of Germany's biggest financial institution on Android 12
2. Type in a username/password field

### Expected behaviour
Open ASK in normal/password mode and enable to type as normal.

### Actual behaviour
When intending to type in the username/password fields, it opens a very limited build-in keyboard instead of ASK. Furthermore, Gboard (factory default) and SwiftKey are supported by that app and behave like expected.

**Android OS version:** 12

**Device manufacturer and model:** Google Pixel 6

**List of installed add-ons (like languages, or themes):**
+ German Language Pack

**Comment by Zoomaaa** [comment]
similar issue here. al Etihad bank app opens for a second and closes itself immediately when ASK is chosen. if i go to another app and switch to samsung default keyboad the app works normally.
[link to the bank app](https://play.google.com/store/apps/details?id=com.ofss.fcdb.mobile.android.phone.BAE.launcher&hl=en_US&gl=US)

**Comment by menny** [comment]
I was not able to use Etihad app, it crashes on start up.
And I can't read German, and don't know what is the flow to get to the username / password page.

Can you provide help with that?

**Comment by swssl** [comment]
Of course i can; After finishing the installation, open the authentication app and tab on "Jetzt einrichten" ("Start setup"), then it asks you for the registration letter you get from your bank. For testing purpose, tab on "Registrierungsbrief erhalten?" ("Received registration letter?"). Afterwards the app displays a text field and wants you to choose a app-password to prevent other persons from opening the app on your device. This is where the described behaviour occurs the first time, but it continues when trying to login into your bank account, where the build-in fallback keyboard does not support all allowed symbols.
- Hope this is helpful

**Comment by Zoomaaa** [comment]
> I was not able to use Etihad app, it crashes on start up.

Yes, this is what happens if the ASK is enabled. Temporarily changing back to Samsung keyboard (I do it quickly on the messenger app) enables you to open the app.
